builtins.go

```diff
@@ -750,43 +834,6 @@ func builtinPow(i *interpreter, trace TraceElement, basev value, expv value) (va
 	return makeDoubleCheck(i, trace, math.Pow(base.value, exp.value))
 }
 
-func builtinSplitLimit(i *interpreter, trace TraceElement, strv, cv, maxSplitsV value) (value, error) {
```

Comment:
Why drop this?

Reply:
eh, looks like a rebase artifact, thanks for catching it

Reply:
fixed